static/app/components/deprecatedDropdownMenu.spec.tsx

```diff
@@ -134,10 +134,10 @@ describe('dropdownMenuDeprecated', function () {
     );
 
     // Make sure this is only called when menu is open
-    expect(addSpy).not.toHaveBeenCalled();
+    expect(addSpy).not.toHaveBeenCalledWith('click', expect.anything(), true);
 
     await userEvent.click(screen.getByRole('button'));
-    expect(addSpy).toHaveBeenCalled();
+    expect(addSpy).toHaveBeenCalledWith('click', expect.anything(), true);
```

Comment:
Unrelated?

Reply:
This test was failing since the `useHotKeys` hook adds the event listener for all tests. I changed it to only look for the event listener it intends to test.

Reply:
oh can you just enable the hotkey when it's open?

Reply:
I don't think so, since the hotkey is on the provider now. If i moved it back to the panel maybe, but otherwise we can't call the hook conditionally IIRC.

Reply:
we could just add a `disabled` option to `useHotKeys` but let's not worry about it for now maybe